### ISSUE TITLE
Config attributes cannot be empty strings in pods

### ISSUE THREAD

**Issue by antoniogamiz** [rakudo/rakudo#3098]
## The Problem

Empty configuration attributes cannot be defined:

~~~perl6
=begin pod :config("")
=end pod
~~~

## Expected Behavior

That code should compile.

## Actual Behavior

This errors is thrown:

~~~
===SORRY!===
No such method 'Str' for invocant of type 'NQPMu'
~~~

## Steps to Reproduce

Execute:

~~~
perl6 -e "=begin pod :kind('') \n =end pod" 
~~~

## Environment
* Operating system: Ubuntu 16.04
* Compiler version (`perl6 -v`): 

~~~
This is Rakudo version 2019.03.1 built on MoarVM version 2019.03
implementing Perl 6.d.
~~~

**Comment by tbrowder** [comment]
I don't agree that an empty value for a config key should be valid, but I do agree that the error message is LTA and should be improved.

**Comment by antoniogamiz** [comment]
Mmm, you're right, an empty value should not be accepted. I will close this, thanks for opening the other issue.

**Comment by taboege** [comment]
Why is an empty value not allowed? If there a prior spec that says so (or a source code comment in rakudo that makes it clear), can you link it? Otherwise my opinion is that this is an edge case bug in the compiler.

Next, what about empty arrays and hashes? The former produces the same error but the latter works:

``` perl6
=begin pod :kind{}
=end pod

dd $=pod.first.config;
#= Hash %!config = {:kind(${})}
```

**Comment by antoniogamiz** [comment]
Mm, if it works with {} it should also work with (). I will reopen to continue the discussion.
